c2corg_ui/static/js/documentediting.js

```diff
@@ -171,6 +171,8 @@ app.DocumentEditingController = function($scope, $element, $attrs,
   if (this.auth_.isAuthenticated()) {
 
     if (this.id_ && this.lang_) {
+      // check each n seconds if user has logged out in between
+      this.recheckAuth_();
```

Comment:
Why not doing it also when creating a new document?

And honestly I would be rather pissed off if I was redirected  to the auth page while editing a doc (losing my work?).